### PR TITLE
wso2-apim simplified setup

### PR DESCRIPTION
**Pull request by NishikaDeSilva**

### CODE REVIEW

File: simple/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,3479 @@
...
+        UM_USER_TENANT_ID INTEGER NOT NULL,
+        UM_ROLE_TENANT_ID INTEGER NOT NULL,
+        UNIQUE(UM_USER_ID,UM_ROLE_ID,UM_USER_TENANT_ID, UM_ROLE_TENANT_ID),
+        FOREIGN KEY(UM_ROLE_ID,UM_ROLE_TENANT_ID) REFERENCES UM_ROLE(UM_ID,UM_TENANT_ID) ON DELETE CASCADE,
+        FOREIGN KEY(UM_USER_ID,UM_USER_TENANT_ID) REFERENCES UM_USER(UM_ID,UM_TENANT_ID) ON DELETE CASCADE
+    )ENGINE INNODB;
+
+    CREATE TABLE UM_ACCOUNT_MAPPING(
+    	UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+    	UM_
```
Comment (chirangaalwis) [inline simple/deployment.yaml]: @NishikaDeSilva Can you please update the Kubernetes Deployment API version as in [1]?

[1]: [Kubernetes API Reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#deployment-v1-apps)

Comment (chirangaalwis) [inline simple/deployment.yaml]: @NishikaDeSilva Please refer the previous comment on updating API version.

Comment (chirangaalwis) [inline simple/deployment.yaml]: @NishikaDeSilva Please refer the previous comment on updating API version.

Comment (chirangaalwis) [inline simple/deployment.yaml]: @NishikaDeSilva Please add Kubernetes resource requests and limits [1] for pods as per product team recommendations.

[1]: [Kubernetes official documentation](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container)

Comment (chirangaalwis) [inline simple/deployment.yaml]: @NishikaDeSilva add new EOF line.

File: simple/getcreds.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```
Comment (chirangaalwis) [inline simple/getcreds.sh]: @NishikaDeSilva Add license header.

File: simple/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,3479 @@
+#kubernetes namespaces
```
Comment (chirangaalwis) [inline simple/deployment.yaml]: @NishikaDeSilva Add license header.

Comment (NishikaDeSilva) [reply simple/deployment.yaml]: is there anything to be changed in the license header i.e: year ?

File: simple/deployment.yaml (hunk 1 above)
Comment (NishikaDeSilva) [reply simple/deployment.yaml]: This one is a group "extension" it's latest is v1beta1. Is it ok if I change the group to app?

Comment (NishikaDeSilva) [reply simple/deployment.yaml]: please refer to comment on https://github.com/wso2/kubernetes-apim/pull/162/files/79a955dad89701fd2ef9e1ec0278217422429dd6#r266561611

File: simple/getcreds.sh (hunk 2 above)
Comment (NishikaDeSilva) [reply simple/getcreds.sh]: Please refer to comment in https://github.com/wso2/kubernetes-apim/pull/162/files/79a955dad89701fd2ef9e1ec0278217422429dd6#r266556306

**Pull request closed without merging**